index.js

```diff
@@ -68,7 +68,7 @@ function exit() {
 }
 
 globals.setImmediate(function () {
-	var numberOfTests = runner.tests.concurrent.length + runner.tests.serial.length;
+	var numberOfTests = runner.tests.tests.concurrent.length + runner.tests.tests.serial.length;
```

Comment:
Could use `.concurrent` and `.serial` in the collection as well, not sure it needs the `tests` container.